nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RMetadataInvalidationEvent.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.r.internal;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Event dispatched when R metadata (PACKAGES files) should be invalidated and rebuilt.
+ */
+public class RMetadataInvalidationEvent
```

Comment:
Please add 'since 1.next' value

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27